HISTORY.rst

```diff
@@ -2,6 +2,12 @@
 History
 =======
 
+0.7.0 (Unreleased)
+==================
+
+* Released with ALFAsim 1.8.0.
+* Add support for WellConectionPort on CaseDescription
```

Comment:
Where did this come from? I believe the support is there since 0.6.0.

Reply:
Nops, the release was first made with the content on ALFAsim.
After that, the modification of WellConnectionPort was introduced.

![image](https://user-images.githubusercontent.com/5083518/98724805-08c0db00-2373-11eb-800a-7de53bd4dde8.png)

Reply:
Hmm nevermind then. The feature itself was done earlier (merged to master 2 days before the tag 0.6.0 was created), but since you were already working on the migration it unsynced, but no big deal.